pkg/apis/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.22
 require (
 	github.com/rancher/wrangler/v2 v2.1.4
 	k8s.io/api v0.28.6
-	k8s.io/apimachinery v0.29.2
+	k8s.io/apimachinery v0.28.6
```

Comment:
:thinking: why a downgrade here, instead of an update to 0.29.4? Is that version not available?
How likely is a version mismatch between `pkg` and the rest of the repo to cause issues?

Reply:
Oh, this makes more sense, thanks for clarifying.